src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    public String value;
+    protected boolean isPrivate;
+
+    public Contact() {
+        super();
```

Comment:
`super()` should be optional here, since you do not need to call the parent class (Object) to initialize anything. You can initialize your variables here if you want.